paasta_tools/utils.py

```diff
@@ -841,7 +842,7 @@ def _now():
 
 def remove_ansi_escape_sequences(line):
     """Removes ansi escape sequences from the given line."""
-    return no_escape.sub('', line)
+    return no_escape.sub('', line) if type(line) is str else no_escape_b.sub(b'', line)
```

Comment:
If the actual runtime only presents bytes, the tests are all currently irrelevant (and incorrect) -- this function should only support one sequence type (either `str` **or** `bytes`).